tools/elfutils/Makefile

```diff
@@ -43,24 +43,44 @@ PKG_GNULIB_MODS = \
 	fallocate-posix \
 	fnmatch-gnu \
 	fts \
-	obstack \
+	obstack-printf-posix \
 	progname \
 	strchrnul \
 	tsearch
 
+HOST_BUILD_PARALLEL:=1
+
 include $(INCLUDE_DIR)/host-build.mk
 
 export $(PKG_GNULIB_BASE)=$(HOST_BUILD_DIR)/$(PKG_GNULIB_BASE)/.libs/$(PKG_GNULIB_BASE).a
+
+export $(PKG_GNULIB_BASE)_exitfail=$(HOST_BUILD_DIR)/$(PKG_GNULIB_BASE)/$(PKG_GNULIB_BASE)_la-exitfail.o
 export $(PKG_GNULIB_BASE)_fallocate-posix=$(HOST_BUILD_DIR)/$(PKG_GNULIB_BASE)/$(PKG_GNULIB_BASE)_la-posix_fallocate.o
+export $(PKG_GNULIB_BASE)_obstack=$(HOST_BUILD_DIR)/$(PKG_GNULIB_BASE)/$(PKG_GNULIB_BASE)_la-obstack.o
+export $(PKG_GNULIB_BASE)_obstack-printf=$(HOST_BUILD_DIR)/$(PKG_GNULIB_BASE)/obstack_printf.o
+export $(PKG_GNULIB_BASE)_printf-args=$(HOST_BUILD_DIR)/$(PKG_GNULIB_BASE)/printf-args.o
+export $(PKG_GNULIB_BASE)_printf-frexp=$(HOST_BUILD_DIR)/$(PKG_GNULIB_BASE)/$(PKG_GNULIB_BASE)_la-printf-frexp.o
+export $(PKG_GNULIB_BASE)_printf-frexpl=$(HOST_BUILD_DIR)/$(PKG_GNULIB_BASE)/$(PKG_GNULIB_BASE)_la-printf-frexpl.o
+export $(PKG_GNULIB_BASE)_printf-parse=$(HOST_BUILD_DIR)/$(PKG_GNULIB_BASE)/printf-parse.o
 export $(PKG_GNULIB_BASE)_tsearch=$(HOST_BUILD_DIR)/$(PKG_GNULIB_BASE)/$(PKG_GNULIB_BASE)_la-tsearch.o
+export $(PKG_GNULIB_BASE)_vasnprintf=$(HOST_BUILD_DIR)/$(PKG_GNULIB_BASE)/vasnprintf.o
 
 HOST_MAKE_FLAGS += \
+	am__CONFIG_DISTCLEAN_FILES= \
 	DEFAULT_INCLUDES='-I. -I$$$$(top_builddir) -I$$$$(top_srcdir)/$(PKG_GNULIB_BASE)' \
 	AM_LDFLAGS='$$$$(STACK_USAGE_NO_ERROR)' \
 	LIBS+='$$$$(if $$$$(findstring $(lastword $(PKG_SUBDIRS)),$$$$(subdir)), $$$$($(PKG_GNULIB_BASE)))' \
```

Comment:
@httpstorm try again like this

`DEFAULT_INCLUDES='-iquote . -I$$$$(top_builddir) -I$$$$(top_srcdir)/$(PKG_GNULIB_BASE)' \`

Reply:
@mcprat Alas I'm getting the same error.

Reply:
can I see the log from this second try?

Reply:
wait try this instead

`DEFAULT_INCLUDES='-iquote . -iquote $$$$(top_builddir) -I$$$$(top_srcdir)/$(PKG_GNULIB_BASE)' \`

Reply:
[2024-11-09.02-elfutils-stack.txt](https://httpstorm.com/share/.openwrt/test/2024-09-28_elfutils/2024-11-09.02-elfutils-stack.txt)

Reply:
well this is silly... the upstream fix is incomplete...

Reply:
from src/Makefile

```
srcdir = .
...
...
...
AM_CPPFLAGS = -iquote . -I$(srcdir) -I$(top_srcdir)/lib -I.. \
...
...
```

Reply:
Yes, I have that pattern too. Can you update the PR or send me diff and I'll give it another test tomorrow.